### ISSUE TITLE
Problem with scrolling a large list

### ISSUE THREAD

**Issue by LongKick64** [petrroll/PowerSwitcher#4]
Set the scroll speed in the system (http://take.ms/1R4a0) does not affect the scrolling in the list (http://take.ms/XHyNW)
Also not right working scroll the list by pressing Page Up and Page Down on the keyboard. Scrolling should be paged, not as pressing Home and End keys.

**Comment by petrroll** [comment]
Yep, I know about both of these issues. Unfortunately I don't really have much time at the moment and I don't think many people will have so many schemas that either of these problems would cause serious usability issues.

Feel free to fiddle with XAML, fix it, and submit PR though :).

**Comment by petrroll** [comment]
Unfortunately I've stopped development of this app so I (most probably) won't be looking at sorting this personally. Feel free to contribute a PR/create a fork, will be happy to help with that.

More info in #23
